extension/chrome/src/js/userhosts.js

```diff
@@ -1,27 +1,35 @@
 document.getElementById('save_user_hosts').addEventListener('click', (event) => {
-	
-	let hostsButton = document.getElementById('user_defined_hosts'),
-		hosts = hostsButton.value.
+
+    let hostsTextarea = document.getElementById('user_defined_hosts');
+    let hosts = hostsTextarea.value.
 		replace(/,/g, ' ').
 		replace(/;/g, ' ').
-		replace(/\r/g, ' ').
-		replace(/\t/g, ' ').
-		replace(/\n/g, ' ').
-		split(' ');
-
-	let fhosts = [];
+		replace(/\t+/g, ' ').
+		replace(/\r+/g, ' ').
+        replace(/\n+/g, ' ').
```

Comment:
Can this be changed to just one replace?
```js
.replace(/[\n\t,;\s\r]+/g, ' ');
```

Reply:
Yep! I added the `+` so it grabs any recurring dividers.